src/Button/IconButton.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import {EyeClosedIcon, EyeIcon, SearchIcon, XIcon, HeartIcon} from '@primer/octicons-react'
+import {Story, Meta} from '@storybook/react'
+import React from 'react'
+import {IconButton} from '.'
+
+const unset = undefined
+const icons = {unset, EyeClosedIcon, EyeIcon, SearchIcon, XIcon, HeartIcon}
+
+export default {
+  title: 'Components/IconButton',
+  argTypes: {
+    size: {
+      control: {
+        type: 'radio',
+        options: ['small', 'medium', 'large']
+      }
+    },
+    disabled: {
+      control: {
+        type: 'boolean'
+      }
+    },
+    variant: {
+      control: {
+        type: 'radio',
+        options: ['default', 'primary', 'danger', 'invisible']
+      }
+    },
+    icon: {
+      control: {
+        type: 'select',
+        options: Object.keys(icons)
+      },
+      mapping: icons
+    },
+    ariaLabel: {control: 'text'}
```

Comment:
Can someone help me figure this prop out? I guess we don't expose it as a normal prop like `ariaLabel`, the consumer is mean to supply `aria-label="hello"`.

- Do we like that API?
- If yes, how can I hook into it to surface it as a Storybook control? Its not camel case..

<img width="1330" alt="error message about storybook prop" src="https://user-images.githubusercontent.com/18661030/194592944-a884cb64-a999-49c9-b1b2-af7161b1bd59.png">

Reply:
From what I observe, for the aria attributes we seem to follow the native convention for our API. `aria-label` over `ariaLabel` 

To make that type error go, you can use `aria-label` instead of `ariaLabel`

For argTypes
```
'aria-label': {control: 'text'}
```

For args
```
'aria-label': 'Icon button description',
```

And I think you don't need to explicitly set up the `aria-label` prop in the component anymore. 

```
<IconButton {...args} />
```

Let me know what you think 👀

Reply:
Thank you! This totally worked 🎉 I didn't know I could use a string for a prop name in Storybook args!